src/js/components/molecules/filter-category/filter-category.vue.html

```diff
@@ -0,0 +1,15 @@
+<!-- @format -->
+
+<section class="filter-category">
+    <h4>{{ title }}</h4>
+    <form>
+        <ul class="filter-category__filter-list" ref="filters" @checkboxChange="handleFilterChange">
+            <li><input class="filter-category__reset" :class="resetActive()" type="reset" :value="showAllLabel" @click="resetFilters" /></li>
```

Comment:
If the `<input type="reset" />` is kept out of the `<ul>`, the for loop can just iterate over the `<li>`s without the need for a `<template>`

Reply:
Also… `resetActive()` should probably be a getter with a slightly clearer name. It didn't make much sense until I saw the template.

Reply:
i don't want to place the reset `<li>` outside of the `<ul>`; i want to keep the html valid.

does `toggleActive` make it clearer?